### PR TITLE
Update templates for https URL

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
@ekohl This dropped my license implementation and just adds a check to set `http://github.com` to `https://github.com`

File: gem2rpm/foreman_plugin.spec.erb [hunk 1]
```diff
@@ -48,7 +48,7 @@ Summary: <%= spec.summary.gsub(/\.$/, "") %>
 Group: Applications/Systems
 License: <%= spec.licenses.empty? ? 'FIXME' : spec.licenses.map { |l| LICENSES.fetch(l, l) }.join(" and ") %>
 <% if spec.homepage -%>
-URL: <%= spec.homepage %>
+URL: <%= spec.homepage.include?('http://github.com') ? spec.homepage.gsub('http://', 'https://') : spec.homepage %>
```
Comment (ekohl) [inline gem2rpm/foreman_plugin.spec.erb]: I wonder if `spec.homepage` is ever `nil`. Other than that I still like fixing it in upstream gemspecs to improve the ecosystem a bit.
```suggestion
URL: <%= spec.homepage.start_with?('http://github.com') ? spec.homepage.gsub('http://', 'https://') : spec.homepage %>
```

**Comment by ehelms** [comment]
I agree fixing it in upstream would be a nice contribution on top of working around those that are not fixed up yet.

**Comment by ekohl** [comment]
If we silently fix it up, we'll never notice. Given how little we regenerate the specs and it's nice to catch on new specs, I'm slightly leaning to leaving this out.

**Comment by ehelms** [comment]
I forsee us updating specs in the future more than you might think. But we can leave this out. I think it's going to bite folks time and time again. And there is no guarantee that every project will accept a fix and release that fix in a timely manner.

**Comment by ekohl** [comment]
I'm almost wondering if we should have a helper library where we centralize this. We've copied the SCL prefixes, licenses and homepage. If we have a helper, we can also introduce an env var to opt in to fix it up.

**Pull request closed without merging**